### PR TITLE
Rename to DotNetBuild.props for unified-build

### PR DESCRIPTION
**Pull request by ViktorHofer**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/10863)

### CODE REVIEW

File: eng/DotNetBuild.props [hunk 1]
```diff
@@ -1,9 +1,10 @@
-<!-- Whenever altering this or other Source Build files, please include @dotnet/source-build-internal as a reviewer. -->
+<!-- Whenever altering this or other DotNetBuild files, please include @dotnet/source-build-internal as a reviewer. -->
```
Comment (ViktorHofer) [inline eng/DotNetBuild.props:1]: @MichaelSimons @mmitche should we change the GH team as well?

Comment (MichaelSimons) [reply eng/DotNetBuild.props:1]: Yes, I think we should since the scope has changed.

Comment (ViktorHofer) [reply eng/DotNetBuild.props:1]: What's the right group to use?

Comment (mmitche) [reply eng/DotNetBuild.props:1]: The file will still contain things interesting for source-only builds, so it should be a group that can encompass the full scope of Unified Build. Maybe such a thing doesn't exist in GH yet.

Comment (ViktorHofer) [reply eng/DotNetBuild.props:1]: Let's keep the existing group for now and merge this in but file an issue to track updating the group.

**Review by JeremyKuhne — APPROVED** [review]

**Pull request merged**